### ISSUE TITLE
Allow task output to be used in the next step

### ISSUE THREAD

**Issue by eduardomourar** [OlafConijn/AwsOrganizationFormation#54]
We need to use the output from previous task (CloudFormation stack) into following tasks, for the cases where we don't know in advance and have to reference .

**Comment by OlafConijn** [comment]
proposal on how to support this in a tasks file.

``` yaml
TaskA:
  Type: update-stacks
  Template: ./templateA.yml. # has export named ExportA
  StackName: stack-a

TaskB:
  DependsOn: TaskA
  Type: update-stacks
  Template: ./templateB.yml
  StackName: stack-b
  Parameters:
    parameterA1: !ImportValue ExportA
    parameterA2: !ImportValue [ExportA, accountId, region]
```

accountId and region are optional. if *not provided* the accountId and region are used where the template is being executed (so each account/region the template gets updated).

Question:
is it more explicit/elegant to **require** something like:

``` yaml
TaskA:
  Type: update-stacks
  Template: ./templateA.yml. # has export named ExportA
  StackName: stack-a
  Export:

```

**Comment by eduardomourar** [comment]
I would suggest to change it to `CopyValue`, being different the behavior

**Comment by OlafConijn** [comment]
first version published to `0.9.5-beta.1`.

example:

``` yaml
BucketTemplate:
  Type: update-stacks
  Template: ./bucket.yml
  StackName: scenario-export-bucket
  DefaultOrganizationBindingRegion: eu-west-1
  DefaultOrganizationBinding:
    IncludeMasterAccount: true

PolicyTemplate:
  DependsOn: BucketTemplate
  Type: update-stacks
  Template: ./bucket-policy.yml
  StackName: scenario-export-bucket-role
  DefaultOrganizationBindingRegion: eu-west-1
  DefaultOrganizationBinding:
    IncludeMasterAccount: true
  Parameters:
    bucketArn: !CopyValue BucketArn
    bucketArn2: !CopyValue [BucketArn, 123123123123]
    bucketArn3: !CopyValue [BucketArn, 102625093955, 'eu-west-1']
```

- `bucketArn` will always look for the export in the account and region it is deploying the stack `scenario-export-bucket-role` to.
- `bucketArn2` will always look for the export in account 123123123123 and  the region it is deploying the stack `scenario-export-bucket-role` to.
- `bucketArn3` will always look for the export in account 123123123123 and eu-west-1.

**Comment by OlafConijn** [comment]
released as part of v0.9.5
